reset-pass/account-validator-tool/src/webapp/css/newUser.css

```diff
@@ -35,6 +35,26 @@ p, h1, form, button {
 	width:50em;
 }
 
+.stylized .topheading {
+    text-align: center;
+    margin-bottom: 12px;
+}
+
+.stylized .logo {
+    background: #444 url(/library/skin/morpheus-default/images/logo.png) no-repeat center center;
```

Comment:
As far as I can see one would need to convert the css files to Sass in order to do this, taking me several days which Adam can't give me at the moment.  If you know a simpler way do let me know!

I've also noticed that there is duplicate javascript, html and css between this module (reset-pass) and the user module so combining these seems a good idea but that is more work.

These fixes do improve the look, are you able to merge them?